controllers/eventController.js

```diff
@@ -12,7 +13,11 @@ const createEvent = asyncHandler(async (req, res, next) => {
 });
 
 const getAllEvents = asyncHandler(async (req, res, next) => {
-  const events = await Event.find({});
+  console.log(req.query.limit);
+  const { page = 1, limit = 4 } = req.query
```

Comment:
Make the limit ```10``. Making it 4 events cannot fill a section.

Reply:
Noted! 
Remove the comments too

Reply:
I have increased the limit to 10

Reply:
> Noted!
> Remove the comments too

which comments?

Reply:
Console.log...

Reply:
> Console.log...

done

Reply:
But how do I take back the .env file
Meanwhile I have added it to .gitignore but idk why it is still being sent

Reply:
I have updated the .gitignore but the .env is still being sent

Reply:
**git rm .env --cached**

Reply:
You have to remove the env from cached and add to  .gitignore

Reply:
after running that command then i tried to push and I got the error

$ git push
To https://github.com/muslimbuilders/halaqaat-server.git
 ! [rejected]        feature/enable-pagination -> feature/enable-pagination (non-fast-forward)
error: failed to push some refs to 'https://github.com/muslimbuilders/halaqaat-server.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

Reply:
then i did git pull and then i tried to push and still got the error